newIDE/app/src/MainFrame/index.js

```diff
@@ -1507,6 +1508,15 @@ const MainFrame = (props: Props) => {
           i18n: i18n,
           project: state.currentProject,
           onChooseProject: chooseProject,
+          onOpenRecentFile: (fileMetadata: FileMetadata) => {
+            openFromFileMetadata(fileMetadata).then(state => {
+              if(state)
+                openSceneOrProjectManager({
+                currentProject: state.currentProject,
+                editorTabs: state.editorTabs,
+                });
+            })
+          }, 
```

Comment:
This is the function I need to call through `ipcRenderer.on('event-name', props.onOpenRecentFile)` but I couldn't figure out how to pass the `fileMetadata`. Need some help :/